manifests/params.pp

```diff
@@ -1,15 +1,15 @@
 # A list of the parameters and their default values for RSyslog.
 #
-# @param service_name [String]
```

Comment:
These should remain params to have the proper documentation format.

See https://tickets.puppetlabs.com/browse/PDOC-153